### PR TITLE
YC-883: Add objects list in emails

### PR DESCRIPTION
**Pull request by rbovard**
Fix https://jira.liip.ch/browse/YC-883

I also added the types list in mail subject https://github.com/yverdon/geocity/pull/408 where it was missing.

### CODE REVIEW

File: geocity/apps/permits/models.py [hunk 1]
```diff
@@ -940,6 +940,16 @@ def get_works_type_names_list(self):
             )
         )
 
+    def get_works_object_names_list(self):
+
+        return ", ".join(
+            list(
+                self.works_object_types.all()
+                .values_list("works_object__name", flat=True)
```
Comment (monodo) [inline geocity/apps/permits/models.py:948]: Use new shortname field ?

File: geocity/apps/permits/services.py [hunk 2]
```diff
@@ -1907,6 +1913,10 @@ def get_works_type_names_list(permit_request):
     return permit_request.get_works_type_names_list()
 
 
+def get_works_object_names_list(permit_request):
+    return permit_request.get_works_object_names_list()
+
+
```
Comment (monodo) [inline geocity/apps/permits/services.py:1919]: Please remove. If you can pass `permit_request `to this method, you can call directly the method on the `permit_request `instance

File: geocity/apps/permits/models.py (hunk 1 above)
Comment (rbovard) [reply geocity/apps/permits/models.py:948]: > Use new shortname field ?

No, as specified here https://jira.liip.ch/browse/YC-883 we want the object name (we already have the type name in the email subject)

> How does the title gets cropped if many WOT are selected ?

Like the type names in the email subject: concatenated and separated by commas.

File: geocity/apps/permits/services.py (hunk 2 above)
Comment (rbovard) [reply geocity/apps/permits/services.py:1919]: So also for `def get_works_type_names_list(permit_request):` I guess?

**Comment by rbovard** [comment]
@monodo Review addressed, please take a look.

**Review by monodo — APPROVED** [review]

**Pull request merged**